src/jailer/src/cgroup.rs

```diff
@@ -449,20 +446,25 @@ pub mod test_util {
     use std::io::Write;
     use std::path::{Path, PathBuf};
 
-    use super::PROC_MOUNTS;
+    use utils::rand;
+
+    pub fn get_mock_proc_mounts() -> String {
+        format!(
+            "/tmp/firecracker/test/{}/jailer/proc/mounts",
+            rand::rand_alphanumerics(4).into_string().unwrap()
+        )
+    }
 
     #[derive(Debug)]
-    pub struct MockCgroupFs {
+    pub struct MockCgroupFs<'a> {
         mounts_file: File,
+        pub proc_mount_path: &'a str,
```

Comment:
nitpicking: the name `proc_mounts_path` is used for `CgroupBuilder` and the file path `/proc/mounts`, so `proc_mounts_path` would be preferable here as well.
```suggestion
        pub proc_mounts_path: &'a str,
```

Reply:
fixed in a9979eda67064f70c1b9f249692b9c0f485d36a1